AdventOfCode.Core/Day7.fs

```diff
@@ -29,23 +29,23 @@ module Day7 =
 
         static let colour =
             [
-                "light red", LightRed
-                "dark orange", DarkOrange
-                "bright white", BrightWhite
-                "muted yellow", MutedYellow
-                "shiny gold", ShinyGold
-                "dark olive", DarkOlive
-                "vibrant plum", VibrantPlum
-                "faded blue", FadedBlue
-                "dotted black", DottedBlack
+                "light", "red", LightRed
+                "dark", "orange", DarkOrange
+                "bright", "white", BrightWhite
+                "muted", "yellow", MutedYellow
+                "shiny", "gold", ShinyGold
+                "dark", "olive", DarkOlive
+                "vibrant", "plum", VibrantPlum
+                "faded", "blue", FadedBlue
+                "dotted", "black", DottedBlack
             ]
-            |> List.map (fun (name, x) -> !& name =% x)
+            |> List.map (fun (style, name, x) -> !& style .>> name =% x)
             |> (||=) "BagColour"
 
-        static let bagWord = regexString "bag(s)?" |> terminal "BagWord" (T(fun _ data -> data.ToString()))
+        static let bagWord = regexString "bag(s)?" |> terminalU "BagWord"
```

Comment:
The 'terminalU` function is part of the [_Untyped API_](https://teo-tsirpanis.github.io/Farkle/advanced-features.html#The-untyped-API).